hub_mozita.py

```diff
@@ -26,23 +28,45 @@ def fix_username(username):
         username = "@" + username
     return username
 
-
+def safe_conf_get(config_parser, section, key_name):
+    '''
+    returns parsed value if key_name exists in config.ini, null otherwise
+    '''
+    try:
+        return config_parser.get(section, key_name)
+    except Exception:
+        print(key_name + " non presente nella sezione " + section + "!")
+        exit()
+
+######################
+#  LOADING SECRETS   #
+######################
+# managing config.ini
 if not os.path.isfile("config.ini"):
     print(
         "Il file di configurazione non è presente.\n" +
-        "Rinomina il file 'config-sample.ini' in 'config.ini' e inserisci il token.".encode("utf-8"))
+        "Rinomina il file 'config-sample.ini' in 'config.ini' e inserisci i dati mancanti.".encode("utf-8"))
     exit()
 
+# useful object to manage secret values
 script_path = os.path.dirname(os.path.realpath(__file__))
 config_parser = ConfigParser()
 config_parser.read(os.path.join(script_path, "config.ini"))
+localtime = datetime.now()
+data_salvataggio = localtime.strftime("%Y_%m_%d")
+    
+###########################
+#  MANAGING BOT CONSTANTS #
+###########################
+TOKEN = safe_conf_get(config_parser, "bot", "TOKEN")
+NEWS_CHANNEL = safe_conf_get(config_parser, "bot", "NEWS_CHANNEL")
```

Comment:
A che cosa si riferisce? (perché non c'è un "NEWS_CHANNEL" nel file config-sample.ini)

Reply:
Eh tocca aggiungerlo nel sample, l'ho scordato.
È il canale ovviamente dove fare il forwarding da Twitter al canale destinazione il nome deve iniziare  con @.

Ricordo inoltre che in quel canale il bot deve avere accesso ai messaggi ed essere admin.

Mi sono scordato di aggiornare il config-sample, scusate!